### DISCUSSION TITLE
[Enhancement] Queue up the artist top songs section automaticly

### DISCUSSION THREAD

**Discussion by gms10ur** [ciderapp/Cider#665] [Feature Request]
**Is your feature request related to a problem? Please describe.**
 I'm always frustrated when I have to add the popular songs in the queue by hand when discovering artists

**Describe the solution you'd like**
when you play a song from artist page popular songs section, the songs underneath should be added to the queue automatically. But the full list of the popular songs, not the lesser view.

**Describe alternatives you've considered**
if the shuffle is on, the popular songs list should be added fully and randomized too. 

**Additional context**
lastly, when the popular songs queue ends, auto queue can initiate to be like the artist radio too

**Comment by elliotjarnit** [comment]
I can try to implement this

**Comment by gms10ur** [comment]
that would be so cool. As a developer myself (mainly c++/{Python) I will look into this stuff when I have time. just not my area of knowledge (yet) :)
